### PR TITLE
Update Slint Community License

### PR DESCRIPTION
**Pull request by aurindam**
Changes:

1. Rename the file to a generic name "Royalty-free" instead of "Royalty-Free-1.1"
2. Rename the license to "Slint Royalty-free Desktop, Mobile, and Web Applications License"
3. Add definition of "Mobile Application" and grant of right under #1. Grant of Rights
4. Move "Limitations" to 3rd section and "License Conditions - Attributions" to 2nd section
5. Allow flexibility to choose between showing "MadeWithSlint" as a dialog/splash screen or on a public webpage
6. Move the para on copyright notices to section under "Limitations"

### CODE REVIEW

File: LICENSES/LicenseRef-Slint-Community.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+# Slint Royalty-free Desktop, Mobile, and Web Applications License
+
+## Preamble
+
+Slint is a toolkit that can be used to build user interfaces for applications. Slint (hereafter referred to as "Software") is made available under different licenses by SixtyFPS GmbH incorporated at Oranienburger Str. 44, 16540 Hohen Neuendorf, Germany ("SixtyFPS"). The "Slint Royalty-free Desktop, Mobile, and Web Applications License" is suitable for those who develop desktop, mobile, or web applications and do not want to use open source components under copyleft licenses.
+
+### 1. Grant of Rights
+
+SixtyFPS hereby grants You a world-wide, royalty-free, non-exclusive license to use, reproduce, make available, modify, display, perform, distribute the Software as part of a Desktop or Web Application.
```
Comment (ogoffart) [inline LICENSES/LicenseRef-Slint-Community.md:11]: should this mention Mobile?

**Review by ogoffart — COMMENTED** [review]
What's the reason to change the name.
We've tried to explain the license with the "royalty-free" name in previous discussions with the community. I'd rather not change the name unless there is a very good reason

File: LICENSES/LicenseRef-Slint-Community.md [hunk 2]
```diff
@@ -1,32 +1,38 @@
-# Slint Royalty-free Desktop and Web Applications License, version 1.1
+Community License, version 1.2
+
+# Slint Royalty-free Desktop, Mobile, and Web Applications License
 
 ## Preamble
 
-Slint is a toolkit that can be used to build user interfaces for applications. Slint (hereafter referred to as "Software") is made available under different licenses by SixtyFPS GmbH incorporated at Oranienburger Str. 44, 16540 Hohen Neuendorf, Germany ("SixtyFPS"). The "Slint Royalty-free Desktop and Web Applications License" is suitable for those who develop desktop or web applications and do not want to use open source components under copyleft licenses.
+Slint is a toolkit that can be used to build user interfaces for applications. Slint (hereafter referred to as "Software") is made available under different licenses by SixtyFPS GmbH incorporated at Oranienburger Str. 44, 16540 Hohen Neuendorf, Germany ("SixtyFPS"). The "Slint Royalty-free Desktop, Mobile, and Web Applications License" is suitable for those who develop desktop, mobile, or web applications and do not want to use open source components under copyleft licenses.
```
Comment (tronical) [inline LICENSES/LicenseRef-Slint-Community.md:7]: > The "Slint Royalty-free Desktop, Mobile, and Web Applications License" is suitable for those

This phrase of the license text suggests to me that either the license to continue to be named as it is now, or this sentence should be adjusted.

File: LICENSE.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
-<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial -->
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.2 OR LicenseRef-Slint-commercial -->
 
 # Slint License
 
 You can use Slint under ***any*** of the following licenses, at your choice:
 
-1. [Royalty-free license](LICENSES/LicenseRef-Slint-Royalty-free-1.1.md),
+1. [Royalty-free license](LICENSES/LicenseRef-Slint-Royalty-free.md),
```
Comment (tronical) [inline LICENSE.md:7]:
```suggestion
1. [Royalty-free license](LICENSES/LicenseRef-Slint-Royalty-free-1.2.md),
```

File: README.md [hunk 4]
```diff
@@ -185,7 +185,7 @@ configure your favorite editor to work well with Slint.
 
 You can use Slint under ***any*** of the following licenses, at your choice:
 
-1. [Royalty-free license](LICENSES/LicenseRef-Slint-Royalty-free-1.1.md),
+1. [Royalty-free license](LICENSES/LicenseRef-Slint-Royalty-free.md),
```
Comment (tronical) [inline README.md:188]:
```suggestion
1. [Royalty-free license](LICENSES/LicenseRef-Slint-Royalty-free-1.2.md),
```

**Pull request merged**